django/pyrog/api/views.py

```diff
@@ -59,6 +59,8 @@ class FilterViewSet(viewsets.ModelViewSet):
     serializer_class = serializers.FilterSerializer
 
 
-class OwnerViewSet(viewsets.ModelViewSet):
+class OwnerViewSet(mixins.CreateModelMixin, mixins.ListModelMixin, mixins.DestroyModelMixin, viewsets.GenericViewSet):
```

Comment:
I think there's nothing in `Owner` that we would want the user to update. `name` is determined by what it receives from credential and `schema` is generated. I'll add `RetrieveModelMixin` although we may not need it since we currently always fetch the entire list of owners for a source.